connexion/decorators/produces.py

```diff
@@ -76,7 +77,12 @@ def dumps(data):
         """ Central point where JSON serialization happens inside
         Connexion.
         """
-        return "{}\n".format(json.dumps(data, indent=2))
+        if six.PY2 == True:
+            encoding = "utf-8"
+        else:
+            encoding = None
+
+        return "{}\n".format(json.dumps(data, indent=2, encoding=encoding))
```

Comment:
@LeenaBhegade I assume it was a typo and @rafaelcaricio meant "{}\n".format(...)